docs/hugo_stats.json

```diff
@@ -317,7 +317,6 @@
       "debugging-pubsub-tests",
       "deduplicator",
       "delay-metadata",
-      "delay-on-error",
```

Comment:
Why this change?

Reply:
this is some garbage generated by hugo, I gitignored it